frontend/src/components/code-editor/helpers.ts

```
@@ -0,0 +1,78 @@
+  toDOM() {
+    return document.createTextNode('⚠');
+  }
+};
+
+export const breakpointState = StateField.define<RangeSet<GutterMarker>>({
+  create() {
+    return RangeSet.empty;
+  },
+  update(set, transaction) {
+    let updatedSet = set.map(transaction.changes);
+    for (let e of transaction.effects) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'e' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)